src/Mailgun/Model/Suppression/Unsubscribe/Unsubscribe.php

```diff
@@ -15,49 +15,101 @@
 class Unsubscribe
 {
     /**
+     * The unsubscribe event identifier
+     * 
      * @var string
      */
-    private $address;
+    private $id = NULL;
+    
+    /**
+     * The unsubscribe email address
+     * @var string
+     */
+    private $address = NULL;
 
     /**
+     * Tag to unsubscribe from, use * to unsubscribe address from domain
+     * 
+     * Note: If NULL the value means * (all for a domain)
+     * 
      * @var string
      */
-    private $tag;
+    private $tag = NULL;
 
     /**
+     * The creation date for the event
+     * 
      * @var \DateTime
      */
-    private $createdAt;
+    private $createdAt = NULL;
 
     /**
      * @param string $address
      */
-    private function __construct($address)
+    private function __construct($address = NULL)
     {
         $this->address = $address;
         $this->createdAt = new \DateTime();
     }
 
     /**
-     * @param array $data
+     * Create a Unsubscribe object based in info provided in $data array
+     * 
+     * Fields allowed:
```

Comment:
Good. Do you know of the PSR5 syntax?

https://github.com/php-http/cache-plugin/blob/master/src/CachePlugin.php#L50-L62

Reply:
Oww! Awesome. I will update it. I didn't know about PSR5